tests/config/utils/check-parsers.js

```diff
@@ -3,9 +3,7 @@
 const path = require("path");
 const { outdent } = require("outdent");
 
-// [prettierx] TBD ???:
-// const TESTS_ROOT = path.join(__dirname, "../../format");
-const TESTS_ROOT = path.join(__dirname, "../../tests");
+const TESTS_ROOT = path.join(__dirname, "../../format");
```

Comment:
Reverting faffaa7bdd8d038cbf3a92dd5bae109bb923a04f seems to lead to this error:

```console
FAIL tests/format/flow/standard/jsfmt.spec.js
  ● Test suite failed to run

    Parser "babel" should not used in "/home/runner/work/prettierx/prettierx/tests/format/flow/standard".

    Suggest move your tests to:
    - /home/runner/work/prettierx/prettierx/tests/format/js
    - /home/runner/work/prettierx/prettierx/tests/format/jsx

    Or config to allow use this parser in "/home/runner/work/prettierx/prettierx/tests/config/utils/check-parsers.js".

      171 |           `;
      172 |
    > 173 |     throw new Error(
          |           ^
      174 |       `Parser "${parser}" should not used in "${dirname}".${
      175 |         suggestion ? `\n\n${suggestion}` : ""
      176 |       }`

      at checkParser (tests/config/utils/check-parsers.js:173:11)
      at runSpec (tests/config/format-test.js:219:5)
      at Object.<anonymous> (tests/format/flow/standard/jsfmt.spec.js:1:83)
```

@adalinesimonian I leave it up to you if you want to investigate this or should consider reapplying my workaround in faffaa7bdd8d038cbf3a92dd5bae109bb923a04f.

Reply:
No problem, I'll take a look.

Reply:
I verified that the functionality in `check-parsers.js` is working as intended - it was simply not checking anything until the directory was corrected to `../../format`. I made tests respect the parser settings in adalinesimonian@53ed3ce. Overview from the commit message:

- HTML void tag test broken out to angular/html/vue
- flow standard test using typescript parser removed, made redundant by adding additional code from flow test to typescript test
- typescript space-before-function-paren test moved to typescript folder
- fixed order of parsers